charts/fleet-simulator/Chart.yaml

```diff
@@ -10,3 +10,5 @@ annotations:
   catalog.cattle.io/namespace: fleet-simulator-system
   catalog.cattle.io/release-name: fleet-simulator
   catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux,windows
```

Comment:
missing node selectors and tolerations to work in windows: https://github.com/rancher/fleet/blob/master/charts/fleet-simulator/templates/deployment.yaml